simpler_setup/scene_test.py

```
@@ -167,6 +167,89 @@ def tensor_names(self) -> list[str]:
+        self._originals.clear()
+        while self._buffers:
+            buf, view = self._buffers.pop()
+            del view
+            try:
+                buf.buffer.release()
+            except (ValueError, BufferError):
+                pass
+            try:
+                self._worker.free_host_buffer(buf)
+            except BaseException:  # noqa: BLE001
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)